lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -21,6 +21,12 @@
 options:
   displayname:
     description: Display name
+  update_password:
+    description:
+    - Set password for a user.
```

Comment:
This needs `type: str`. Also please move `version_added` below `choices`.

Reply:
Ah I didn't see the `type:` in any of the other options.

All changed.